docs/start/quickstart.md

```diff
@@ -1 +1,192 @@
 # Quickstart
+
+The programming model consists of wrapping the code containing the model training logic inside a function. 
+Inside that wrapper function provide all imports and parts that make up your experiment.
+
+## What can I do with Maggy?
+
+Maggy can be used for three different ML experiments: Distributed Hyperparameter Optimization (HPO), 
```

Comment:
four different

data parallel training, and model-parallel training (with DeepSpeed Zero)